ArtOfIllusion/src/artofillusion/script/ExecuteScriptWindow.java

```diff
@@ -15,7 +15,21 @@
 import buoy.event.*;
 import buoy.widget.*;
 import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
 import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.prefs.Preferences;
```

Comment:
Generally, if you're importing this many classes from a single package you might as well just wildcard `import java.util.*;`